pkgs/applications/video/kodi/default.nix

```diff
@@ -223,11 +223,6 @@ in stdenv.mkDerivation {
     # I'm guessing there is a thing waiting to time out
     doCheck = false;
 
-    postPatch = ''
-      substituteInPlace xbmc/platform/linux/LinuxTimezone.cpp \
-        --replace 'usr/share/zoneinfo' 'etc/zoneinfo'
-    '';
-
```

Comment:
Does upstream provide the proper path now?

Reply:
Upstream does not have that file anymore

Reply:
The file is still there from what I see, with no recent changes:
https://github.com/xbmc/xbmc/blob/Leia/xbmc/platform/linux/LinuxTimezone.cpp
I think you removed the wrong patch. The one that's no longer needed is the one in patches attr, as the comment suggests

Reply:
@cpages yes, sorry my bad... I removed the wrong patch... not I fixed it